coinstacks/bitcoin/api/src/models.ts

```diff
@@ -1,25 +1,123 @@
 /* unable to import models from a module with tsoa */
-import { Account } from '../../../common/api/src'
+import { Account, Tx } from '../../../common/api/src'
 
 /**
- * Contains additional bitcoin specific info
+ * Contains info about a transactions ScriptSig
  */
-export interface BitcoinAccount extends Account {
-  /**
-   * Account details by address if BitcoinAccount was fetched by xpub
-   */
-  addresses?: Array<Account>
+export type ScriptSig = {
+  asm: string
+  hex: string
+}
+
+/**
+ * Contains info about a transactions ScriptPubKey
+ */
+export type ScriptPubKey = {
+  asm: string
+  hex: string
+  reqSigs: number
+  type: string
+  addresses: string[]
+}
+
+/**
+ * Contains info about a transactions inputs
+ */
+export type Vin = {
+  txid?: string
+  vout?: number
+  sequence?: number
+  n?: number
+  addresses?: Array<string>
+  isAddress?: boolean
+  value?: string
+  hex?: string
+  asm?: string
+  coinbase?: string
+  scriptSig?: ScriptSig
+}
+
+/**
+ * Contains info about a transactions outputs
+ */
+export type Vout = {
+  value?: string
+  n: number
+  spent?: boolean
+  spentTxId?: string
+  spentIndex?: number
+  spentHeight?: number
+  hex?: string
+  asm?: string
+  addresses: Array<string> | null
+  isAddress: boolean
+  type?: string
+  scriptPubKey?: ScriptPubKey
+}
+
+/**
+ * Contains detailed info about a Bitcoin transaction
+ */
+export type BtcTxSpecific = {
+  txid: string
+  hash?: string
+  version?: number
+  size?: number
+  vsize?: number
+  weight?: number
+  locktime?: number
+  vin: Array<Vin>
+  vout: Array<Vout>
+  hex?: string
+  blockhash?: string
+  confirmations: number
+  time?: number
+  blocktime?: number
 }
 
 /**
  * Contains info about an unspent transaction output
  */
 export interface Utxo {
-  address: string
-  confirmations: number
   txid: string
-  value: string
   vout: number
+  value: string
+  height?: number
+  confirmations: number
+  address?: string
+  path?: string
+  locktime?: number
+  coinbase?: boolean
+}
+
+/**
+ * Contains additional bitcoin specific info
+ */
+export interface BitcoinAccount extends Account {
+  /**
+   * Account details by address if BitcoinAccount was fetched by xpub
+   */
+  addresses?: Array<Account>
+
+  /**
+   * Transactions of the given xpub
+   */
+  transactions?: Array<Tx>
```

Comment:
This can be removed with no longer requesting `txs` details from getAccount

Reply:
mostly agree I think. is it worth including txids? concern that would cause a burden for other chain implementations?